src/ChunkedStreamDecoder.php

```diff
@@ -87,6 +87,14 @@ protected function iterateBuffer()
         }
 
         if ($this->nextChunkIsLength) {
+            if (substr($this->buffer, 0, 3) === "0\r\n") {
```

Comment:
Tests already check for that: https://github.com/WyriHaximus/http-client/blob/26f43a22e23cc8dc879bbee0c392bbd56b1ce023/tests/DecodeChunkedStreamTest.php#L145 But does it could be a bit better, lets try to get that into `v0.4.16`, I'll get this out the door in `v0.4.15`.